planning/mission_planner/src/mission_planner_lanelet2/mission_planner_lanelet2.cpp

```diff
@@ -295,6 +320,7 @@ autoware_auto_planning_msgs::msg::HADMapRoute MissionPlannerLanelet2::planRoute(
   route_msg.segments = route_sections;
   route_msg.goal_pose = goal_pose_.pose;
 
+  RCLCPP_WARN(get_logger(), "Goal Pose Z : %lf", goal_height);
```

Comment:
I prefer to make this RCLCPP_DEBUG. (I know it was easier for you to work with WARN during development though)